src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -337,5 +346,38 @@ private static ILogger CreateLogger(bool verbose, ReportFormat reportFormat, IRe
 
             return loggerFactory.CreateLogger("TemplateAnalyzerCLI");
         }
+
+        private void SummarizeLogs(bool verbose)
+        {
+            if (loggedErrors.Count > 0 || loggedWarnings.Count > 0)
+            {
+                Console.ForegroundColor = ConsoleColor.Red;
```

Comment:
I think errors should be in red, but warnings in yellow.

Reply:
Maybe other text that isn't an error/warning message in the default console color?

Reply:
I left it red when there are errors, and yellow when there are only warnings. What do you think of that? I think it might be important to also highlight those lines (the verbose message, the count of errors, etc.)

Reply:
Works for me.